Jenkinsfile_CNP

```diff
@@ -67,7 +67,8 @@ def secretsForPR = [
                 secret('docmosis-endpoint', 'PDF_SERVICE_CONVERT_URL'),
                 secret('docmosis-api-key', 'PDF_SERVICE_ACCESS_KEY'),
                 secret('AppInsightsInstrumentationKey', 'APPINSIGHTS_INSTRUMENTATIONKEY'),
-                secret('sscs-servicebus-connection-string-tf', 'SERVICE_BUS_CONNECTION_STRING')
+                secret('sscs-servicebus-connection-string-tf', 'SERVICE_BUS_CONNECTION_STRING'),
+                secret('preview-ccd-db-password-2', 'PREVIEW_DB_PASSWORD_2')
```

Comment:
Not a show stopper, but I don't see why we are setting only one API password like this ? where others apps are reading from elsewhere. We only create one db server for all apps in preview, so ideally everything should use same way, a maintenance overhead if not.

Reply:
For WA we were told to use the Azure PostgreSQL flexible servers, that password is in the vault but this is the only way I could override the default password for the am-role-assignment-service. This can be removed when we update to newer CCD chart.

Reply:
Ahh sure, you don't need them in vault. they are setup as secrets in flux. Application can read like in https://github.com/hmcts/chart-ccd/pull/278/files#diff-5c4fad46c3e248239dcb1170b473d88ff66da225f099d0b6f897d710c1a0fb13R116-R119 . With having it vault, they will be in 2 places and likely go out of sync at some point.

Reply:
It's encrypted as a secret here : https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/preview/sops-secrets/sscs-postgres.enc.yaml

Reply:
Basically what u removed should work. https://github.com/hmcts/sscs-tribunals-case-api/pull/3793/files#diff-23620b4ffeebecff87d5e2a077b1d0afb80739578e0999577266b0e4cc19b093L335-L339

Reply:
@adusumillipraveen the bit I removed worked on the newer CCD chart but didn't work on the older one. It was still using the password from the ccd chart.

Reply:
This is an additional maintenance though when upgrading, may be leaving a note to remove it on upgrade.